editor/editor_properties_array_dict.cpp

```diff
@@ -483,9 +483,10 @@ bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
 	}
 
 	Dictionary drag_data = p_drag_data;
+	const String drop_type = drag_data.get("type", "");
 
-	if (drag_data.has("type") && String(drag_data["type"]) == "files") {
-		Vector<String> files = drag_data["files"];
+	if (drop_type == "files") {
+		Array files = drag_data["files"];
```

Comment:
```suggestion
		PackedStringArray files = drag_data["files"];
```
That one was correct.